installing release-it v20 produces peer deps errors

installing recent release-it v20 produces npm peer dependency errors 

```sh
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: native-copyfiles@2.0.1
npm warn Found: release-it@19.2.4
npm warn node_modules/release-it
npm warn   dev release-it@"^20.0.0" from the root project
npm warn   1 more (@release-it/conventional-changelog)
npm warn
npm warn Could not resolve dependency:
npm warn peer release-it@"^18.0.0 || ^19.0.0" from @release-it/conventional-changelog@10.0.6
npm warn node_modules/@release-it/conventional-changelog
npm warn   dev @release-it/conventional-changelog@"^10.0.6" from the root project
```

it's caused by the strict peer deps

https://github.com/release-it/conventional-changelog/blob/344bad3bf2f90618a86d3e38883a5d5eff12aaf1/package.json#L52-L54

--- a/package.json
+++ b/package.json
@@ -44,13 +44,13 @@
   },
   "devDependencies": {
     "installed-check": "^9.3.0",
-    "release-it": "^19.2.4",
+    "release-it": "^20.0.0",
     "shelljs": "^0.10.0",
     "sinon": "^21.0.2",
     "tmp": "^0.2.5"
   },
   "peerDependencies": {
-    "release-it": "^18.0.0 || ^19.0.0"
+    "release-it": "^18.0.0 || ^19.0.0 || ^20.0.0"
   },
   "engines": {
     "node": "^20.12.0 || >=22.0.0"

